src/librustc_codegen_ironox/builder.rs

```diff
@@ -352,35 +394,40 @@ impl BuilderMethods<'a, 'tcx> for Builder<'a, 'll, 'tcx> {
     )-> <Self::CodegenCx as BackendTypes>::Value {
         unimplemented!("array_alloca");
     }
+
     fn load(
         &mut self,
         ptr: <Self::CodegenCx as BackendTypes>::Value,
         align: Align
     )-> <Self::CodegenCx as BackendTypes>::Value {
         unimplemented!("load");
     }
+
     fn volatile_load(
         &mut self,
         ptr: <Self::CodegenCx as BackendTypes>::Value
     )-> <Self::CodegenCx as BackendTypes>::Value {
         unimplemented!("volatile_load");
     }
+
     fn atomic_load(
         &mut self,
         ptr: <Self::CodegenCx as BackendTypes>::Value,
         order: AtomicOrdering, size: Size
     )-> <Self::CodegenCx as BackendTypes>::Value {
         unimplemented!("atomic_load");
     }
+
     fn range_metadata(
         &mut self,
         load: <Self::CodegenCx as BackendTypes>::Value,
         range: Range<u128>
     ) {
         unimplemented!("range_metadata");
     }
+
     fn nonnull_metadata(&mut self, load: <Self::CodegenCx as BackendTypes>::Value) {
-        unimplemented!("nonnull_metadata");
```

Comment:
Did you accidentally remove an error message here?

Reply:
Good spot! Fixed